c.md

```diff
@@ -238,6 +238,12 @@ if(conditional-expression)
     //code
 }
 ```
+### 2.1. If-else using Ternary Operator
+```c
+conditional-expression ? code1 : code2;
+```
+If conditional-expression is true, code1 is executed. And, if condition is false, code2 is executed.
```

Comment:
Instead this, can you add self explanatory code block? like below 

`bool isEven = (i % 2 == 0) ? true : false;`

Reply:
Yes, surely
can I go with the same example as mentioned?
@onecompiler-ops

Reply:
Yes, please test once and add it